pypeit/core/procimg.py

```diff
@@ -958,7 +930,7 @@ def variance_frame(datasec_img, sciframe, gain, ronoise, darkcurr=None,
             Dark current in electrons per second if the exposure time is
```

Comment:
I'd say maybe let's just require the dark current to always be in electrons per second, and thus require that both dark current and exposure time be set. But I'd have to see how we are using this. My guess is that we are actually never taking dark current into account.